app/views/rubygems/_rubygem.html.erb

```diff
@@ -1,10 +1,10 @@
-<%= link_to rubygem_path(rubygem), :class => 'gems__gem' do %>
+<%= link_to "/gems/#{rubygem.name}", :class => 'gems__gem' do %>
```

Comment:
You can pass in the param instead of the activerecord object, so: `rubygem_path(rubygem.name)`.